src/app/pages/partners/partner-details/partner-details.js

```diff
@@ -171,14 +171,14 @@ function PartnerDetails({model}) {
 
     return (
         <div
-            className="partner-details" onClick={(e) => e.stopPropagation()}
+            className="partner-details"
             ref={ref}
         >
             <div className="sticky-region">
                 <Synopsis {...{model, icon, partnerLinkProps}} />
                 <TabGroup {...{labels, selectedLabel, setSelectedLabel}} />
             </div>
-            <div className="scrolling-region boxed" onClick={(e) => e.stopPropagation()}>
```

Comment:
@RoyEJohnson i have no idea what these `stopPropagation` were here for, but they make it real hard to integrate with what is happening. do you know what their purpose is?

Reply:
I don't know specifically, but I would do this if clicks were being captured on something that contained these things. It's been in there since Nov 2020. My guess is it was event-tracking related and it will not introduce problems to take it out.